src/quo2/components/list_items/user_list/style.cljs

```diff
@@ -0,0 +1,40 @@
+(ns quo2.components.list-items.user-list.style
+  (:require [quo2.foundations.colors :as colors]))
+
+(defn user-list-wrapper [can-be-invited? background-type]
+  (let [height (if can-be-invited? 56 80)]
+    {:width              359
+     :border-radius      12
+     :padding-vertical   8
+     :padding-horizontal 12
+     :height             height
+     :background-color   (case background-type
+                           :5  colors/primary-50-opa-5
+                           :10 colors/primary-50-opa-10
+                           :transparent :transparent)
```

Comment:
I find it odd to read that `background-type` can be a keyword `:5` (as a number), it almost looks as a typo even though it might not be.

It'd be nice if we could map `:5` and `:10` to non-magical values. Is there any semantic meaning to them? That can give you a clue about their *type*.

Also note that `case` throws an exception without a default case.

Reply:
Maybe `opacity-10` ?